src/i18n.js

```diff
@@ -92,6 +95,7 @@ const I18NManager = ({languageSelectorTarget, children}) => {
 
 I18NManager.propTypes = {
   languageSelectorTarget: PropTypes.instanceOf(Element),
+  onLanguageChangeDone: PropTypes.instanceOf(Function),
```

Comment:
```suggestion
  onLanguageChangeDone: PropTypes.func,
```
see https://reactjs.org/docs/typechecking-with-proptypes.html